src/message/messageActionSheet.js

```diff
@@ -39,6 +45,13 @@ const reply = ({ message, dispatch, auth }) => {
 };
 reply.title = 'Reply';
 
+const replyWithMention = ({ message, dispatch, auth }) => {
+  const narrow = getNarrowFromMessage(message, auth.email);
+  dispatch(doNarrow(narrow, message.id));
+  dispatch(draftUpdate(narrow, `@**${message.sender_full_name}** `));
```

Comment:
Hmm, this looks fragile if it works at all -- we narrow, and only *then* we set the draft? Consider how and when the draft gets used.

Also, how does this behave in the case where we're already in the relevant narrow?